src/safeds/data/image/containers/_image.py

```diff
@@ -23,71 +30,106 @@ class Image:
 
     Parameters
     ----------
-    data : BinaryIO
-        The image data as bytes.
+    image_tensor : Tensor
+        The image data as tensor.
     """
 
+    _pil_to_tensor = PILToTensor()
+    _default_device = torch.device("cuda") if torch.cuda.is_available() else torch.device("cpu")
+
     @staticmethod
-    def from_jpeg_file(path: str | Path) -> Image:
+    def from_file(path: str | Path, device: Device = _default_device) -> Image:
```

Comment:
The `device` parameter leaks implementation details. A novice should not have to know that an image is stored internally as a tensor and that the tensor can be stored on different devices. Remove this parameter, please and call `_get_device` (see above) instead.

Reply:
see #524